sample/scripts/initFormatBar.ts

```diff
@@ -45,6 +46,17 @@ export default function initFormatBar() {
         toggleNumbering(getCurrentEditor());
     });
 
+    // Blockquote
+    document.getElementById('blockquoteButton').addEventListener('click', function() {
+        toggleBlockQuote(getCurrentEditor(), (element: HTMLElement) => {
```

Comment:
Make this param optional. We should not make this API too hard to use.

Reply:
Changed

Reply:
Removed these values. Added default styler in toggleBlockQuote()